junifer/preprocess/confounds.py

```diff
@@ -147,7 +210,7 @@ def validate_input(self, input: List[str]) -> None:
             If the input does not have the required data.
 
         """
-        _required_inputs = ["BOLD", "confounds"]
+        _required_inputs = ["BOLD", "BOLD_confounds"]
```

Comment:
Is there a way we don't have to override the method?

Reply:
No, this requires "all of" and not "any of" like the super method.

Reply:
I see.